packages/tslint-config-loris/index.js

```diff
@@ -0,0 +1,115 @@
+module.exports = {
+    // Add additional rules from https://github.com/buzinas/tslint-eslint-rules
+    'rulesDirectory': 'node_modules/tslint-eslint-rules/dist/rules',
+    'rules': {
+        // TypeScript specific
+        'no-empty-interface': true,
+        'no-internal-module': true,
+        'typedef-whitespace': [
+            true,
+            {
+                'call-signature': 'nospace',
+                'index-signature': 'nospace',
+                'parameter': 'nospace',
+                'property-declaration': 'nospace',
+                'variable-declaration': 'nospace'
+            }
+        ],
+        'array-type': [true, 'array'],
+        'class-name': true,
+
+        // https://github.com/buzinas/tslint-eslint-rules#possible-errors
+        'trailing-comma': [true, {multiline: 'never', singleline: 'never'}],
+        'no-conditional-assignment': true,
+        'no-console': [
+            true,
+            // Need manually specify all disallowed functions.
+            'assert',
+            'dir',
+            'error',
+            'info',
+            'log',
+            'time',
+            'timeEnd',
+            'trace',
+            'warn'
+        ],
+        'no-constant-condition': [true, {checkLoops: false}],
+        'no-control-regex': true,
+        'no-debugger': true,
+        'no-duplicate-case': true,
+        'no-empty-character-class': true,
+        'no-ex-assign': true,
+        'no-extra-semi': true,
```

Comment:
Looks like this rule is covered by [semicolon](https://palantir.github.io/tslint/rules/semicolon/) rule from `tslint`, which is used below.